spec/acceptance/nodesets/ubuntu-16-docker.yml

```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  ubuntu-16-docker:
+    roles:
+      - master
+    platform: ubuntu-1404-x86_64
```

Comment:
I [described the exact problem](https://tickets.puppetlabs.com/browse/BKR-821?focusedCommentId=404039&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-404039) on the Beaker bug.